template/files/README.md

```diff
@@ -0,0 +1,16 @@
+## Development Setup
+ ### Setup the project
+ ```
+git clone <github url of the repo>
+
+bin/setup
+```
+ Update the configuration in  `config/application.yml`.
+ ### Start the server
+ ```
+foreman start
+```
+ ### Run the test suite
+ ```
+bin/rake
+```
```

Comment:
Could you add a placeholder section with `TODO` for the developer to explain how to get a developer API key from the external app. See https://github.com/maestrano/connector-salesforce#configuration or the ones you added for the other connectors.

See the way the original `README.rdoc` is asking the developer to add details how the application.